src/common/gitProvider/bitbucket.ts

```diff
@@ -0,0 +1,226 @@
+import { GitProviderRoot } from "./gitProviderRoot";
+import * as c from "chalk";
+import { PullRequestMessageRequest, PullRequestMessageResult } from ".";
+import { git, uxLog } from "../utils";
+import { Bitbucket, Schema } from "bitbucket";
+
+export class BitbucketProvider extends GitProviderRoot {
+    private bitbucket: InstanceType<typeof Bitbucket>;
+
+    constructor(){
+        super();
+        const token = process.env.CI_SFDX_HARDIS_BITBUCKET_TOKEN;
+        const clientOptions = { auth: { token: token }};
+        this.bitbucket = new Bitbucket(clientOptions);
+    }
+
+    public getLabel(): string {
+        return "sfdx-hardis Bitbucket connector";
+    }
+
+    public async getCurrentJobUrl(): Promise<string> {
+        if (process.env.BITBUCKET_WORKSPACE && process.env.BITBUCKET_REPO_SLUG && process.env.BITBUCKET_BUILD_NUMBER){
+            const jobUrl = `https://bitbucket.org/${process.env.BITBUCKET_WORKSPACE}/${process.env.BITBUCKET_REPO_SLUG}/pipelines/results/${process.env.BITBUCKET_BUILD_NUMBER}`;
```

Comment:
Can the base url of bitbucket be taken from an ENV var ?
In case bitbucket is on premise, not on bitbucket.org :)